recipe/meta.yaml

```diff
@@ -1,32 +1,38 @@
 {% set name = "sentence-transformers" %}
-{% set version = "2.0.0" %}
+{% set version = "2.2.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 492eae8e9973f2b758d66d74325661e2816f0b6637365554a1eca9bcf60088ed
+  sha256: dbc60163b27de21076c9a30d24b5b7b6fa05141d68cf2553fa9a77bf79a29136
 
 build:
+  # skip s390x and ppc64le because torchvision does not support these
+  skip: True  # [linux and (s390x or ppc64le)]
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vvv --no-deps --no-build-isolation
   noarch: python
 
 requirements:
   host:
     - pip
-    - python >=3.6
+    - python
+    - wheel
```

Comment:
Technically requires `setuptools` as well, even though it's probably already installed with another dependency: https://github.com/UKPLab/sentence-transformers/blob/v2.2.2/setup.py#L1

Reply:
thanks!

Reply:
weird that the linter didn't whine about it.

Reply:
Funny, there is actually no lint that checks for setuptools.